lib/git-fastclone.rb

```diff
@@ -343,18 +335,24 @@ def prefetch(submodule_file)
     # that this repo has been updated on this run of fastclone
     def store_updated_repo(url, mirror, repo_name, fail_hard)
       unless Dir.exist?(mirror)
-        Terrapin::CommandLine.new('git clone', '--mirror :url :mirror')
-                             .run(url: url.to_s, mirror: mirror.to_s)
+        fail_pipe_on_error(['git', 'clone', verbose ? '--verbose' : '--quiet', '--mirror', url.to_s, mirror.to_s], quiet: !verbose)
       end
 
-      Terrapin::CommandLine.new('cd', ':path; git remote update --prune').run(path: mirror)
-
+      Dir.chdir(mirror) {
+        output = fail_on_error(*['git', 'remote', verbose ? '--verbose' : nil, 'update', '--prune'].compact, quiet: !verbose)
```

Comment:
if i use `fail_pipe_on_error` it will still spit out a lot of output when verbose is not set
But this works, don't know if it's the best way though